CleanupOptionsBinder.cs

```diff
@@ -0,0 +1,64 @@
+﻿using AMSMigrate.Contracts;
+using System.CommandLine;
+using System.CommandLine.Binding;
+
+namespace AMSMigrate
+{
+    internal class CleanupOptionsBinder : BinderBase<CleanupOptions>
+    {
+        private readonly Option<string> _sourceAccount = new Option<string>(
+             aliases: new[] { "--source-account-name", "-n" },
+             description: "Azure Media Services Account.")
+        {
+            IsRequired = true,
+            Arity = ArgumentArity.ExactlyOne
+        };
+
+        private readonly Option<string?> _filter = new Option<string?>(
+            aliases: new[] { "--resource-filter", "-f" },
+            description: @"An ODATA condition to filter the resources only when the source account is for media service.
+e.g.: ""name eq 'asset1'"" to match an asset with name 'asset1'.
+Visit https://learn.microsoft.com/en-us/azure/media-services/latest/filter-order-page-entities-how-to for more information.")
+        {
+            Arity = ArgumentArity.ZeroOrOne
+        };
+
+        private readonly Option<bool> _isForceCleanUpAsset = new(
+          aliases: new[] {"--force-cleanup", "-x"},
+          () => false,
+          description: @"Force the cleanup of migrated assets if the setting is enabled")
```

Comment:
It might be better to change the description as:

Force the cleanup of the selected input assets no matter what migration status is.